src/components/graphCard/graphCardMetricTotals.js

```diff
@@ -54,134 +52,110 @@ const GraphCardMetricTotals = ({
   const dailyHasData = isCurrent ? currentHasData : lastHasData;
   const dailyValue = isCurrent ? currentValue : lastValue;
 
-  return (
-    <div data-test={`graphMetricTotals-${_camelCase(metricId)}`} className="curiosity-usage-graph__totals">
```

Comment:
required: the `data-test` attributes are used by QE we need to make sure an alternative setup is in place before removing these attributes

Reply:
That is correct, iqe-curiosity-plugin depends on `graphDailyTotalCard` attribute value.

I see below that there's still `data-test`, but value is `graphCard-${index}`. Assuming index evaluates to numerical index, I don't really fancy this idea. I would prefer to have descriptive names - `graphCard-daily`, `graphCard-overage` etc.

Changing string itself is alright, but will require us to change string in iqe plugin, too. We might have a regression run or two failed between the time this is deployed and our automation is fixed, depending on how well we coordinate this.

Reply:
review updates are in
- we're retaining the parent `data-test` attribute... it's located above the cards, line `58` in this set of comments, `graphMetricTotals-${_camelCase(metricId)}``
   - this metricId will get "fuzzy" in the future when more than one metric is displayed in the graph that is dissimilar... ie. Cores vs Sockets. This could be resolved with something like `graphMetricTotals-${_camelCase(metricId-1)}-${_camelCase(metricId-2)}-${_camelCase(metricId-3)}`
- unfortunately, the child cards are still going to be hit if we want to make the cards configurable based on anything ux/design wants in there. Which means the current solution `graphMetricTotals-${index}` is where we're stuck. However, there is one possible alternative we can think of, we're sure there are more out there...
   - we pass in a specific `setting` in product configuration solely for testing purposes... not a big fan of a singular purpose setting, but it is feasible @mirekdlugosz  
   - we are def open to alternative solutions

Reply:
forgot to also note
- the current limit on nested metric cards being planned is `3` to `4`... so `index` would never go past that set point